pontoon/base/templates/translate.html

```diff
@@ -239,27 +239,34 @@ <h3 class="no-match"><div class="fa fa-exclamation-circle"></div>No results</h3>
         <li><a href="#plural6"><span></span><sup></sup></a></li>
       </ul>
     </nav>
-    <textarea dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
-
+    {% if user.is_authenticated() %}
+      <textarea dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
+    {% else %}
+      <textarea readonly dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
+    {% endif %}
```

Comment:
Is there a way to do this ? If I wrap the `readonly` in the `if` clause, for a logged in user, he/she would see 2 text areas, one which is readonly and one which is editable